webapp/components/reftest-analyzer.js

```diff
@@ -180,16 +180,15 @@ class ReftestAnalyzer extends LoadingState(PolymerElement) {
 
     // Set the img srcs manually so that we can promisify them being loaded.
     const imagePromises = ['before', 'after'].map(prop => {
-      if (!this[prop]) {
-        return Promise.reject(`${prop} is empty`);
-      }
-      const img = this.shadowRoot.querySelector(`#${prop}`);
-      const loaded = new Promise((resolve, reject) => {
+      return new Promise((resolve, reject) => {
```

Comment:
Can we simplify this even further to get rid of the curly brackets and return?

Reply:
Yes, can do when at a computer.